index.html

```diff
@@ -18,10 +18,11 @@
   <div class="container page-title">
```

Comment:
You should move this container `div` into the `header`, so that the background color takes the page's full width